### DISCUSSION TITLE
Unjustified Vercel Account Ban - Seeking Advice from the Community

### DISCUSSION THREAD

**Discussion by 4ndual** [vercel/community#2351] [Help]
### Summary

Hey everyone,

I'm reaching out to you all because I'm dealing with an issue with my Vercel account. About a year ago, my account was blocked without a clear explanation. I only had one project deployed on the platform, which I used for personal purposes and to show off my skills to potential employers when I was job hunting.

The website I deployed was just a JSON file that fetched a personal movie list (a list I made in my personal IMDb account) using a third-party app (https://imdb-api.com/). The JSON was then fetched to display the list in my Stremio app. You can check out the code for the project here: https://github.com/4ndual/stremio-list-Addon-/blob/main/index.js

As far as I am aware, I did not violate any of Vercel's terms of service or community guidelines. I used the platform only for personal use and did not engage in any illegal or unethical activities. Despite this, my account was banned, and I recently received a response from Vercel stating, "Your account was blocked for hosting content that is against our TOS. Unfortunately, your account cannot be enabled."

I've looked over their terms of service to make sure I didn't accidentally break any rules, but I can't find any reason for the ban.

I'm asking for advice and help from you all:

Has anyone else been in a similar situation, and if so, how did you sort it out?
Are there any specific steps I should take to escalate this issue with Vercel or get more info on why they think I broke their TOS?
Any other tips or advice that could help me fix this?
I really appreciate any help or suggestions you guys can give. I want to keep using Vercel and hope to resolve this issue soon. Thanks in advance for your help!

**Comment by amyegan** [comment]
Hi @4ndual. You can find more info in our [Fair Use Policy](https://vercel.com/docs/concepts/limits/fair-use-policy) and [Terms of Service](https://vercel.com/legal/terms). If you have more questions, I encourage you to follow up with the help desk for clarification. Do bear in mind that the team may be unavailable to share specific block reasons.
